poll-webapp/src/main/webapp/skin/less/poll.less

```diff
@@ -34,4 +34,128 @@
         padding: 5px 5px;
       }
   }
+}
+
+#poll-activity {
+  .poll-content {
+    font-family: 'Avenir', Helvetica, Arial, sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    color: #2c3e50;
```

Comment:
we should use variables instead for all color codes, check with Houssem which standard colors to be used otherwise create a new one